evap/results/templates/results_evaluation_detail.html

```diff
@@ -72,10 +72,15 @@ <h3>{{ evaluation.full_name }} ({{ evaluation.course.semester.name }})</h3>
     <div class="card card-outline-primary mb-3">
         <div class="card-header d-flex">
             <div class="mr-auto">{% trans 'Overview' %}</div>
+            {% if view == "export" or view == "full" %}
+                <a class="btn btn-sm btn-light d-print-none" href="{% url 'results:evaluation_text_answers_export' evaluation.id %}?view={{ view }}{% if contributor_id is not None %}&contributor_id={{ contributor_id }}{% endif %}" type="button">
+                    {% trans 'Export text answers' %}
+                </a>
+            {% endif %}
```

Comment:
This button is also visible for students (you can confirm it by logging in as “student” with password “evap”). At least they get a 403. Could you change the conditions for the button here?